sentences/nb/homeassistant_HassTurnOn.yaml

```
@@ -9,7 +9,7 @@ intents:
           - "<åpne> <navn>"
           - "<åpne> <navn> <i_på> <område>"
         excludes_context:
-          domain:
+          domain: . binary_sensor
```

I'm guessing this is what you tried to do..?
```suggestion
            - binary_sensor
```